package.json

```diff
@@ -37,11 +37,15 @@
     "file-loader": "^0.8.1",
     "immutable": "^3.7.1",
     "iso": "^4.1.0",
-    "jquery": "^2.1.3",
+    "jquery": "^2.1.4",
+    "jquery-ui": "^1.10.5",
     "jscs-loader": "0.0.5",
     "jsx-loader": "^0.13.1",
     "keymirror": "^0.1.0",
-    "lodash": "^3.7.0",
+    "restify": "^3.0.3",
+    "restify-https": "0.0.3",
+    "fb": "git+https://github.com/Thuzi/facebook-node-sdk.git",
+    "lodash": "^3.9.3",
```

Comment:
Happy to bump the existing versions of jquery, lodash up, however I don't think we will be needing restify, restify-https, fb as of yet. :)